bls/src/main/java/tech/pegasys/teku/bls/BLS.java

```diff
@@ -178,13 +187,18 @@ public static boolean fastAggregateVerify(
       LOG.warn("Skipping bls verification.");
       return true;
     }
-    if (publicKeys.isEmpty()) return false;
-    List<PublicKey> publicKeyObjects =
-        publicKeys.stream().map(BLSPublicKey::getPublicKey).collect(Collectors.toList());
+
     try {
-      return signature.getSignature().verify(publicKeyObjects, message);
-    } catch (DeserializeException e) {
-      return false;
+      if (publicKeys.isEmpty()) return false;
```

Comment:
nit: Braces.

Reply:
Haha, I didn't perpetrate this :-)  But have fixed it and a couple of others.